### PR TITLE
Add documentation for GetFormattedValue

### PR DESCRIPTION
**Pull request by saskliutas**
Added documentation for `GetFormattedValue` in ECExpressions.

### CODE REVIEW

**Review by pmconne — APPROVED** [review]

File: docs/learning/presentation/ECExpressions.md [hunk 1]
```diff
@@ -284,6 +284,7 @@ Symbol                            | Deprecated Symbol               | Type    |
 Symbol                                | Type     | Value
 --------------------------------------|----------|----------
 `Set(number1, number2, ..., numberN)` | number[] | Create a [value list](#value-lists) of the supplied numbers.
+`GetFormattedValue(this.MyProp, "Metric|UsCustomary|UsSurvey|BritishImperial")` | any | Returns property value formatted using specified unit system. If unit system is not specified default presentation units are used to format value
```
Comment (grigasp) [inline docs/learning/presentation/ECExpressions.md:287]: I think `|` needs to be escaped when used inside \`\`.

File: docs/changehistory/NextVersion.md [hunk 2]
```diff
@@ -62,3 +62,10 @@ There is now a method called `lookAtGlobalLocationFromGcs` on [ViewState3d]($fro
 
 * `rootToCartographicFromGcs` behaves like `rootToCartographic` except it is async and uses the GCS to reproject the location.
 * `cartographicToRootFromGcs` behaves like `cartographicToRoot` except it is async and uses the GCS to reproject the location.
+
+## Formatted property values in ECExpressions
```
Comment (grigasp) [inline docs/changehistory/NextVersion.md:66]: Let's make this a sub-topic of `## Presentation` to make it easier to understand the context where this function is availab.e

**Review by grigasp — CHANGES_REQUESTED** [review]

File: docs/learning/presentation/ECExpressions.md (hunk 1 above)
Comment (saskliutas) [reply docs/learning/presentation/ECExpressions.md]: Escaped.

File: docs/changehistory/NextVersion.md (hunk 2 above)
Comment (saskliutas) [reply docs/changehistory/NextVersion.md]: Creted sub-topic

**Review by grigasp — APPROVED** [review]

**Pull request merged**